python/src/FunctionalChaosValidation_doc.i.in

```diff
@@ -0,0 +1,164 @@
+independent observations, which is required to state that the LOO sample MSE
+is an estimator of the true MSE.
+For example, using cross-validation on a quasi-Monte Carlo (QMC) experiment may
+not necessarily provide an accurate estimate of the MSE, because the
+internal structure of the QMC is broken by the different splits:
+the elementary volumes are not filled as expected anymore and the
+space-filling properties of the sequence are lost.
+
+Examples
+--------
+Create a polynomial chaos for the Ishigami function.
+We use the :class:`~openturns.PenalizedLeastSquaresAlgorithmFactory` class
+to specify the method to compute the coefficients.
+We may use the :class:`~openturns.LeastSquaresMetaModelSelectionFactory` class
+in order to create a sparse polynomial chaos expansion, but this
+prevents us from using the fast analytical formulas and get an
+accurate estimator of the MSE.
+
+>>> import openturns as ot
+>>> from math import pi
+>>> from openturns.usecases import ishigami_function
+>>> im = ishigami_function.IshigamiModel()
+>>> sampleSize = 500 
+>>> inputTrain = im.distributionX.getSample(sampleSize)
+>>> outputTrain = im.model(inputTrain)
+>>> multivariateBasis = ot.OrthogonalProductPolynomialFactory([im.X1, im.X2, im.X3])
+>>> selectionAlgorithm = ot.PenalizedLeastSquaresAlgorithmFactory()
+>>> projectionStrategy = ot.LeastSquaresStrategy(inputTrain, outputTrain, selectionAlgorithm)
+>>> totalDegree = 8
+>>> enumerateFunction = multivariateBasis.getEnumerateFunction()
+>>> basisSize = enumerateFunction.getStrataCumulatedCardinal(totalDegree)
+>>> adaptiveStrategy = ot.FixedStrategy(multivariateBasis, basisSize)
+>>> chaosalgo = ot.FunctionalChaosAlgorithm(
+...     inputTrain, outputTrain, im.distributionX, adaptiveStrategy, projectionStrategy
+... )
+>>> chaosalgo.run()
+>>> result = chaosalgo.getResult()
+
+Validate the polynomial chaos expansion using leave-one-out cross-validation.
+
+>>> splitterLOO = ot.LeaveOneOutSplitter(sampleSize)
+>>> validation = ot.FunctionalChaosValidation(result, splitterLOO)
+>>> r2Score = validation.computeR2Score()
+>>> print('R2 = ', r2Score[0])
+R2 =  0.99...
+
+Validate the polynomial chaos expansion using K-Fold cross-validation
+and set the :math:`k` parameter.
+
+>>> kParameter = 10
+>>> splitterKF = ot.KFoldSplitter(sampleSize, kParameter)
+>>> validation = ot.FunctionalChaosValidation(
+...     result, splitterKF
+... )
+
+Draw the validation graph.
+
+>>> graph = validation.drawValidation()"
+
+// ---------------------------------------------------------------------
+
+%feature("docstring") OT::LinearModelValidation::getSplitter
+"Get the cross-validation method.
+
+Returns
+-------
+splitter : :class:`~openturns.SplitterImplementation`
+    The cross-validation method."
+
+// ---------------------------------------------------------------------
+
+%feature("docstring") OT::FunctionalChaosValidation::computePredictivityFactor
```

Comment:
Now that MetaModelValidation has a new name (see [here](https://openturns.github.io/openturns/master/user_manual/response_surface/_generated/openturns.MetaModelValidation.html?highlight=metamodelvalidation#openturns.MetaModelValidation.computeR2Score)), this should be named `computeR2Score`.